Magento_OrderHistorySearch/templates/filters.phtml

```diff
@@ -97,116 +98,226 @@
             info
             extra-order-search
             display-none
+            row
         "
         id="extra-order-search-filters"
     >
-        <div class="row">
+        <div
+            class="
+                field
+                field-order-number
+                field-50
+                col-xs-12
+                col-md-6
+            "
+        >
+            <label
+                class="label"
+                for="order-number"
+            >
+                <?= $escaper->escapeHtml(__('Order Number')); ?></label>
```

Comment:
formatting

Reply:
this was done on purpose, there is extra space when it's formatted different way